setup.py

```diff
@@ -13,4 +13,10 @@
     include_package_data=True,
     classifiers=[],
     python_requires=">=3.6",
+    install_requires=[
+        # By definition, a Custom Component depends on Streamlit.
+        # If your component has other Python dependencies, list
+        # them here.
+        "streamlit >= 0.63",
+    ],
```

Comment:
I neglected to add this to our template before (it's in the updated templates in beta5).